go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -368,29 +378,39 @@ func (collector *TableGC) shouldTransitionTable(tableName string) (shouldTransit
 	return true, state, uuid, nil
 }
 
-// checkTables looks for potential GC tables in the MySQL server+schema.
-// It lists _vt_% tables, then filters through those which are due-date.
-// It then applies the necessary operation per table.
-func (collector *TableGC) checkTables(ctx context.Context, dropTablesChan chan<- string, transitionRequestsChan chan<- *transitionRequest) error {
+// readTables reads the list of _vt_% tables from the database
+func (collector *TableGC) readTables(ctx context.Context) (gcTables []*gcTable, err error) {
```

Comment:
refactored `checkTables()` into `readTables()` + `checkTables()` so that `checkTables()` does not have any DB access and is therefore easy to unit-test.